docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_EfficientDet_Models.md

```diff
@@ -50,7 +50,7 @@ referenced in the **"Pretrained EfficientDet Checkpoints"** section of the model
 
 .. code-block:: sh
 
-    mo --runmode=saved_model --model_name=efficientdet-d4  --ckpt_path=efficientdet-d4 --saved_model_dir=savedmodeldir
+    python3 model_inspect.py --runmode=saved_model --model_name=efficientdet-d4  --ckpt_path=efficientdet-d4 --saved_model_dir=savedmodeldir
```

Comment:
I think we need to re-consider this steps. As I understand, `model_inspect.py` is used to freeze the model from SM format. But we support SM format directly so there is no need to use it and we can skip this step

Reply:
Removed freezing step from this tutorial. 
But currently conversion command works only with 'use_legacy_frontend'.
I created ticket to fix it: 107860